.claude/skills/orchestrate/scripts/classify-pane.sh

```
@@ -0,0 +1,82 @@
+#   tmux-target: e.g. "work:0", "work:1.0"
+#   idle-threshold-seconds: seconds before hash-stable output is declared idle (default: 300)
+#
+# Output (stdout): JSON object:
+#   { "state": "running|idle|waiting_approval|complete", "reason": "...", "pane_cmd": "..." }
+#
+# Exit codes: 0=ok, 1=tmux window not found
+
+set -euo pipefail
+
+TARGET="${1:-}"
+IDLE_THRESHOLD="${2:-300}"
```

Shellcheck (0.11.0)

[warning] IDLE_THRESHOLD appears unused. Verify use (or export if used externally).

(SC2034)